code/game/objects/items/food/bread.dm

```diff
@@ -256,6 +256,68 @@
 	tastes = list("bread" = 1)
 	foodtypes = GRAIN
 	venue_value = FOOD_PRICE_CHEAP
+	/// whether this is in fake swordplay mode or not
+	var/fake_swordplay = FALSE
+
+/obj/item/food/baguette/Initialize()
+	. = ..()
+	register_context()
+
+/obj/item/food/baguette/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	. = ..()
+	if (user.mind?.miming && held_item == src)
+		context[SCREENTIP_CONTEXT_RMB] = "Toggle Swordplay"
+		return CONTEXTUAL_SCREENTIP_SET
+
+/obj/item/food/baguette/examine(mob/user)
+	var/examine_list = ..()
+	if(user.mind?.miming)
+		examine_list += span_notice("You can wield this like a sword by right clicking it.")
+
+/obj/item/food/baguette/attack_self_secondary(mob/user, modifiers)
+	. = ..()
+	if(!user.mind?.miming)
+		return
+	if(fake_swordplay)
+		end_swordplay(user)
+	else
+		begin_swordplay(user)
+
+/obj/item/food/baguette/proc/begin_swordplay(mob/user)
+	visible_message(
+		span_notice("[user] begins wielding [src] like a sword!"),
+		span_notice("You begin wielding [src] like a sword, with a firm grip on the bottom as an imaginary handle.")
+	)
+	attack_verb_continuous = list("slashes", "cuts")
+	attack_verb_simple = list("slash", "cut")
```

Comment:
could you add "touche" to the verb lists here, to go with the standard baguette french fencing joke